sim/midas/src/main/scala/midas/core/SimWrapper.scala

```diff
@@ -72,6 +72,23 @@ trait UnpackedWrapperConfig {
   }
 }
 
+
+/**
+ *  Represents the interface of the target to which bridges connect.
+ */
+trait TargetChannelIO {
```

Comment:
I guess you need to rebase?

Reply:
Should work now, unfortunately chained PRs don't seem to work as expected.